pkgs/by-name/ta/tabby-terminal/package.nix

```diff
@@ -0,0 +1,215 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  git,
+  patch-package,
+  electron_29,
+  prefetch-yarn-deps,
+  fixup-yarn-lock,
+  python3,
+  makeWrapper,
+  http-server,
+  fontconfig,
+  pkg-config,
+  libsecret,
+  desktop-file-utils,
+}:
+
+let
+  tabby = rec {
+    pname = "tabby-terminal";
+    version = "1.0.215";
+
+    src = fetchFromGitHub {
+      owner = "Eugeny";
+      repo = "tabby";
+      rev = "v${version}";
+      hash = "sha256-l3Hyt3e1edJjb7vL66JDllkpHwTU5efsNbgJ5Pcvn60=";
+      leaveDotGit = true;
+      deepClone = true;
+    };
+
+    meta = with lib; {
+      description = "A terminal for a more modern age";
+      homepage = "https://tabby.sh";
+      license = licenses.mit;
+      mainProgram = "tabby";
+      maintainers = with maintainers; [ geodic ];
+      platforms = [ "x86_64-linux" ];
```

Comment:
Arm64 would technically work, but I cannot test it, so I didn't want to include it. Darwin is a little more complicated as setting the icon paths isn't as easy + I cannot test it.

Reply:
You can actually test aarch64-linux by adding it to `boot.binfmt.emulatedSystems` in your system config, and then build with `nix build --system aarch64-linux`. Can't help with Darwin though, since I don't have a Mac either

Reply:
Thanks. Will try...